packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -51,10 +51,14 @@ abstract class CameraPlatform extends PlatformInterface {
   }
 
   /// Creates an uninitialized camera instance and returns the cameraId.
+  ///
+  /// [audioFormatGroup] is used to specify audio codec to be used during recording.
+  /// `null` value will allow platform to select the audio codec.
   Future<int> createCamera(
     CameraDescription cameraDescription,
     ResolutionPreset? resolutionPreset, {
     bool enableAudio = false,
+    AudioFormatGroup? audioFormatGroup,
```

Comment:
This is a breaking change. If we did move forward with this, it would need to follow https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changing-platform-interface-method-parameters